bitnami/phpmyadmin/requirements.yaml

```
@@ -1,4 +1,9 @@
 dependencies:
+  - name: common
+    version: "^0.3.0"
```

You can use:
```suggestion
    version: 0.x.x
```
I'll take the latest `0.x.x` version by default.

---

I can do this, but as the common chart only implement this in version 0.3.0 and higher (assumption), I used this notation to allow everything <2.0.0.

Ref: https://helm.sh/docs/topics/charts/

> Apart from version constrains employing operators `= != > < >= <=` the following shorthand notations are supported
> * hyphen ranges for closed intervals, where 1.1 - 2.3.4 is equivalent to >= 1.1 <= 2.3.4.
> * wildcards x, X and *, where 1.2.x is equivalent to >= 1.2.0 < 1.3.0.
> * tilde ranges (patch version changes allowed), where ~1.2.3 is equivalent to >= 1.2.3 < 1.3.0.
> * **caret ranges (major version changes allowed), where ^1.2.3 is equivalent to >= 1.2.3 < 2.0.0.**

---

It's fine. Let's leave as it's.